scripts/clan.py

```diff
@@ -746,6 +755,11 @@ def load_clan_json(self):
         game.clan.deputy_predecessors = clan_data["deputy_predecessors"]
         game.clan.med_cat_predecessors = clan_data["med_cat_predecessors"]
         game.clan.med_cat_number = clan_data["med_cat_number"]
+        if "custom_pronouns" in clan_data.keys():
+            if clan_data["custom_pronouns"]:
+                game.clan.custom_pronouns = clan_data["custom_pronouns"]
+        else:
+            game.clan.custom_pronouns = []
```

Comment:
I would remove this "else" statement, since custom_pronouns is already set as a empty list in the clan constructor.